website/docs/r/vm_sizing_policy.html.markdown

```diff
@@ -0,0 +1,119 @@
+---
+layout: "vcd"
+page_title: "vCloudDirector: vcd_vm_sizing_policy"
+sidebar_current: "docs-vcd-resource-vm-sizing-policy"
+description: |-
+  Provides a vCloud Director VM sizing policy resource. This can be
+  used to create, modify, and delete VM sizing policy.
+---
+
+# vcd\_vm\_sizing\_policy
+
+Provides a vCloud Director VM sizing policy resource. This can be
+used to create, modify, and delete VM sizing policy.
+
+Supported in provider *v3.0+*
+
+## Example Usage
+
+```hcl
+resource "vcd_vm_sizing_policy" "minSize" {
+  org         = "my-org" # Optional
+  name        = "min-size"
+  description = "smallest size"
+
+  cpu {
+    shares                = "886"
+    limit_in_mhz          = "2400"
+    count                 = "9"
+    speed_in_mhz          = "2500"
+    cores_per_socket      = "3"
+    reservation_guarantee = "0.55"
+  }
+
+  memory {
+    shares                = "1580"
+    size_in_mb            = "3200"
+    limit_in_mb           = "2800"
+    reservation_guarantee = "0.3"
+  }
+}
+```
+## Argument Reference
+
+The following arguments are supported:
+
+* `org` - (Optional) The name of organization to use, optional if defined at provider level. Useful when connected as sysadmin working across different organizations
+* `name` - (Required) The name of VM sizing policy.
+* `description` - (Optional) description of VM sizing policy.
+* `cpu` - (Optional) Configures cpu policy; see [Cpu](#cpu) below for details.
+* `memory` - (Optional) Configures memory policy; see [Memory](#memory) below for details.
+ 
+ <a id="cpu"></a>
+ ## Cpu
+ 
+ Each VM sizing policy supports the following attributes:
+ 
+ * `shares` - (Optional)Defines the number of CPU shares for a VM. Shares specify the relative importance of a VM within a virtual data center. If a VM has twice as many shares of CPU as another VM, it is entitled to consume twice as much CPU when these two virtual machines are competing for resources. If not defined in the VDC compute policy, normal shares are applied to the VM.
+ * `limit_in_mhz` - (Optional) Defines the CPU limit in MHz for a VM. If not defined in the VDC compute policy, CPU limit is equal to the vCPU speed multiplied by the number of vCPUs.
+ * `count` - (Required) Defines the number of vCPUs configured for a VM. This is a VM hardware configuration. When a tenant assigns the VM sizing policy to a VM, this count becomes the configured number of vCPUs for the VM.
```

Comment:
Why is it named `count` instead of `cpus` like in VM? You actually need to look into docs to understand what this `count` means currently.